build/integrationtests/Dockerfile

```diff
@@ -1,23 +1,29 @@
 ARG cycle
 ARG hub
+ARG integrationtests
 
 FROM ${hub}/deploy-env:${cycle}
-
 LABEL maintainer Rob Bovill <rbovill@lsst.org>
+LABEL integrationtests=${integrationtests}
 
-ARG integrationtests
+WORKDIR ${HOME}
 
-RUN source /home/saluser/.setup_sal_env.sh && \
-    conda install -c lsstts -y ts-integrationtests=${integrationtests}
+RUN mkdir Reports
 
-WORKDIR /home/saluser/
+RUN source /home/saluser/.setup_sal_env.sh && \
+mamba install -c lsstts ts-integrationtests=${integrationtests} -y && \
+mamba install -c conda-forge robotframework && \
+pip3 install -U lsst-efd-client
 
-ENV RUN_ARG="simple_script.py"
+RUN git clone https://github.com/lsst-ts/robotframework_EFD.git
 
-LABEL integrationtests=${integrationtests}
+WORKDIR ${HOME}/robotframework_EFD
+RUN ${HOME}/.checkout_repo.sh main develop
```

Comment:
you just cloned the repo a couple lines above, do you need to do this?

Reply:
I guess we'll only want main for the actual executions.  Removed.